src/bench.c

```diff
@@ -11,6 +11,31 @@
 #include "util.h"
 #include "bench.h"
 
+void help(char **argv) {
+    printf("Benchmarks the following algorithms: \n");
+    printf("    1. ECDSA signing/verification\n");
+    printf("    2. ECDH key exchange (optional module)\n");
+    printf("    3. Public key recovery (optional module)\n");
+    printf("    4. Schnorr signatures (optional module)\n");
+    printf("\n");
+    printf("The default number of iterations for each benchmark is 20000. This can be customized\n");
```

Comment:
Text looks nice here! Maybe make the default a variable and then print it here. Then whenever we change the default, the number gets updated automatically in the help text.

Reply:
Nice!

Reply:
I initially thought of creating a macro for the default value in `bench.h`. The other benchmarks (like `bench_ecmlult` uses 10000) use a different default iteration value so, I declared a local variable in the main function for the default value.